cumulusci/tasks/salesforce/sourcetracking.py

```diff
@@ -397,8 +381,7 @@ def _run_task(self):
 
             if changes:
                 self._store_snapshot(changes)
-                maxrevision = max(change["RevisionNum"] or -1 for change in changes)
-                self._store_maxrevision(maxrevision)
+            self._reset_sfdx_snapshot()
```

Comment:
Is this outdent intention? It looks like a possible behavior change, albeit probably not a undesirable one.

Reply:
It's intentional -- probably doesn't matter much, but I figured it'll make sure sfdx gets reset even if there's some error in how cci is keeping track of changes.